releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md

```diff
@@ -1,6 +1,13 @@
 ## Microsoft.Azure.AppConfiguration.AspNetCore
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore)
 
+### 4.0.0-preview - July 23, 2020
+* Added multi-targeting support for .NET Core 3.1 besides .NET Standard 2.0. [#173](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/173)
+* Extracted out the code to retrieve required dependencies for configuration refresh from `IApplicationBuilder.UseAzureAppConfiguration()` to `IServiceCollection.AddAzureAppConfiguration()` in the package `Microsoft.Extensions.Configuration.AzureAppConfiguration`, so it could be re-used for non ASP.NET Core applications.
+    * Users must call `IServiceCollection.AddAzureAppConfiguration()` to register the required services for configuration refresh before they can call `IApplicationBuilder.UseAzureAppConfiguration()`.
```

Comment:
Added a bit more details:

Users must call `IServiceCollection.AddAzureAppConfiguration()` in `ConfigureServices(...)` to register the required services for configuration refresh before they can call `IApplicationBuilder.UseAzureAppConfiguration()`. With this, users can obtain instances of `IConfigurationRefresher` easily from DI in a controller or a middleware and have better control of when and how configuration should be refreshed.